robot.py

```diff
@@ -87,7 +78,11 @@ def robotInit(self):
             if key == 'VISION':
                 self.vision = self.initVision(config)
             if key == 'SWERVOMETER':
+                self.savedSwervometerConfig = config
```

Comment:
I don't think we need to save this reference for later, because we can always get to every value through `self.config["SWERVOMETER"]`.